### PR TITLE
Restricts certain traitor items from appearing in low pop

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Certain items are very difficult for small crews with little or no security to deal with, and lead to one sided murder sprees more often than not.

Adrenals will not appear with less than 30 people

Holoparasites will not appear with less than 30 people

Syndiborgs will not appear with less than 35 people (nuke ops have something like an 80% winrate at this population, I think they'll manage)

### CODE REVIEW

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -63,6 +65,7 @@ var/list/uplink_items = list()
 	var/list/gamemodes = list() // Empty list means it is in all the gamemodes. Otherwise place the gamemode name here.
 	var/list/excludefrom = list() //Empty list does nothing. Place the name of gamemode you don't want this item to be available in here. This is so you dont have to list EVERY mode to exclude something.
 	var/surplus = 100 //Chance of being included in the surplus crate (when pick() selects it)
+	var/player_minimum = FALSE //If 1 or higher, will check that there are enough people online before adding the item to the uplink
```
Comment (RemieRichards) [inline code/datums/uplink_item.dm]: 0 please, not FALSE, this is a numerical value not a boolean.

**Comment by PKPenguin321** [comment]
only thing here that's even remotely sketchy is adrenals being restricted
either way :+1:

**Comment by KorPhaeron** [comment]
@PKPenguin321 

If some guy wants to go killing everyone with the entire armoury with 20 people online he can suck it up and die if someone manages to slip him

**Comment by tkdrg** [comment]
Suggestion: Consider restricting/increased the cost for the same items that gia nerfed on ministation, namely the .357 revolver, ebow, esword, minibomb and noslips.

**Comment by KorPhaeron** [comment]
@tkdrg 

if(I.low_pop_cost)
    I.cost =  I.lowpop cost

Or something like that?

**Comment by tkdrg** [comment]
yep

**Comment by Iamgoofball** [comment]
Rule of thumb is "if the only available gamemode is traitor or changeling, its lowpop"

**Comment by tkdrg** [comment]
Cult is now available for 10+ players, though.

**Comment by Lumipharon** [comment]
You can get infinte adrenals from R&D anyway, if you know what you're doing.
I would honestly increase the borg pop requirement - 35 still isn't that much.

**Comment by AlexanderUlanH** [comment]
>Adrenals lowpop restricted
>People arguing for revolver, ebow, esword to be lowpop restricted

It's exactly as I'd feared.
As long as this doesn't become "I don't want traitors to kill anyone in lowpop ever" it's fine.

**Comment by tkdrg** [comment]
You can get revolver, ebow or esword, you just can't get ebow + esword or revolver + esword

**Comment by optimumtact** [comment]
git rebase -Xignore-all-space upstream/master 
updates this pr cleanly (assuming you have updated upstream/master to the latest)

**Pull request closed without merging**